modules/cold-extraction/ColdDataRetriever.py

```diff
@@ -161,33 +132,35 @@ def initialize():
     subprocess.call("{0}/storescp --accept-unknown --directory {1} --filepath {2} -b {3} > storescp.out &".format(DCM4CHE_BIN, storage_folder, file_path, QUERY_AET), shell=True)
 
 
-
-with open(csv_file, newline='') as f:
-    reader = csv.reader(f)
-    next(f)
-    for row in reader:
-        row = [x.strip() for x in row]
-        if (extraction_type == 'empi_date'):
-            if not ((row[patient_index] == "") or (row[date_index] == "")):
-                patients.append(row[patient_index])
-                temp_date = row[date_index]
-                dt_stamp = datetime.datetime.strptime(temp_date, date_format)
-                date_str = dt_stamp.strftime('%Y%m%d')
-                dates.append(date_str)
-                length = len(patients)
-        elif (extraction_type == 'empi'):
-            if not ((row[patient_index] == "")):
-                patients.append(row[patient_index])
-                length = len(patients)
-        elif (extraction_type == 'accession'):
-            if not ((row[accession_index] == "")):
-                accessions.append(row[accession_index])
-                length = len(accessions)
-        elif (extraction_type == 'empi_accession'):
-            if not ((row[patient_index] == "") or (row[accession_index] == "")):
-                patients.append(row[patient_index])
-                accessions.append(row[accession_index])
-                length = len(accessions)
+def read_csv():
+    global upload_folder
+    upload_file = upload_folder + csv_file # Change the storage folder path
+    with open(upload_file, newline='') as f:
```

Comment:
Here, too it should be just,
with open(csv_file, newline='') as f:

as in line 165.

The rationale is, as explained above.

It is csv_file. We should not call different names just because it is coming from the front-end.